deploy/operator.yaml

```diff
@@ -179,7 +179,7 @@ spec:
   publishNotReadyAddresses: true
   ports:
     - port: 443
-      targetPort: 54874
+      targetPort: 54875
```

Comment:
We randomly generated the original port, are you saying it is taken? Maybe this fixed the issue for you since we failed to cleanup the original allocation of the port.

Reply:
Looks like we were facing collision, I think the best solution (although it will use a litle more memory at masters) is to use DaemonSet for handling Reconcile and Deployment for webhook (so we can run it without host networking and we don't have the port issue)